.circleci/config.yml

```diff
@@ -477,6 +481,15 @@ jobs:
       - store_artifacts:
           path: /tmp/release.tar.gz
       - teardown_vm
+  promote_nightly:
```

Comment:
So cutting nightly is on github actions and promoting nightly is on circleci?

Reply:
Yes because building happens on CircleCI, and we want to make sure the promote PR only comes after the files have been successfully uploaded to storage :)

Reply:
Thinking about this a bit more, I think it's better to have this as a separate job that runs on GitHub Actions. That job can be scheduled to run 20 minutes after the branch cut (as the automated build resulting from the branch cut takes about 10 minutes to finish) and have a safety check that `https://ampjs.org/rtv/${nightlyAmpVersion}/v0.js` exists before performing the update. Will update this PR now